extensions/barrage/src/main/java/io/deephaven/extensions/barrage/BarrageMessageWriterImpl.java

```diff
@@ -108,10 +108,10 @@ public BarrageMessageWriter newMessageWriter(
             return new BarrageMessageWriterImpl(message, chunkWriters, metricsConsumer) {
                 @Override
                 protected void writeHeader(
-                        ByteBuffer metadata,
-                        MutableInt size,
-                        FlatBufferBuilder header,
-                        ExposedByteArrayOutputStream baos) throws IOException {
+                        final ByteBuffer metadata,
```

Comment:
I may have gone overboard making things final to get rid of yellow in my IDE.  If you guys object, I'll revert those changes.